mpi/bugs/iprobe-overflow.c

```diff
@@ -6,9 +6,17 @@ int main(int argc, char* argv[])
 {
     MPI_Init(&argc, &argv);
 
+/*
+*   Icc threw a diagnostic when i declared in for().
```

Comment:
Declaring variables in the loop is standard C99.  I guess ICC defaults to C89/GNU89.